### PR TITLE
updated range slider

### PR DESCRIPTION
**Pull request by alexkhalevine**

### CODE REVIEW

File: src/components/RangeSlider/customTheme.scss [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+  @media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
+    height: 74px;
+    margin-top: -8px;
+  }
+
+  &:focus {
+    position: relative;
+    outline: none;
+
+    &::-webkit-slider-thumb {
```
Comment (codingcircus) [inline src/components/RangeSlider/customTheme.scss]: would this still work, if you just comma separate the three vendor slider-thumbs? They all have the same styling

File: src/components/RangeSlider/icons/icn-decrease-btn.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    height="25px"
+    viewBox="0 0 32 32"
+  >
+    <g
+      fill="none"
+      stroke="none"
+      strokeWidth="1"
+      fillRule="evenodd"
+    >
+      <g transform="translate(-52.000000, -822.000000)">
```
Comment (codingcircus) [inline src/components/RangeSlider/icons/icn-decrease-btn.jsx]: Maybe this svg can still be cleaned up a little bit, I don't think the two `g` tags are required (however, you'd need to check positioning then)

File: src/components/RangeSlider/icons/icn-increase-btn.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    viewBox="0 0 32 32"
+  >
+    <g
+      fill="none"
+      stroke="none"
+      strokeWidth="1"
+      fillRule="evenodd"
+    >
+      <g transform="translate(-9.000000, -822.000000)">
+        <g transform="translate(10.000000, 823.000000)">
```
Comment (codingcircus) [inline src/components/RangeSlider/icons/icn-increase-btn.jsx]: see comment in decrease-btn

File: src/components/RangeSlider/index.scss [hunk 4]
```diff
@@ -174,6 +168,59 @@ $lower-background: linear-gradient(to top, $lower-color, $lower-color) 100% 50%
   }
 }
 
-.sliderErrorContainer {
-  margin: -10px 0 10px 0;
+
+.sliderGridContainer {
+  width: 100%;
+  position: relative;
+  display: grid;
```
Comment (codingcircus) [inline src/components/RangeSlider/index.scss]: When thinking about it, I think this should actually be achieved using flexbox

Comment (codingcircus) [reply src/components/RangeSlider/index.scss]: Then you also need less feature-queries

File: src/components/RangeSlider/customTheme.scss [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+$height: 30px;
+$thumb-height: 25px;
+$track-height: 4px;
+
+$upper-color: lighten($red, 40);
+$lower-color: $blue;
+$thumb-color: $red;
+$thumb-hover-color: $blue;
+
+$upper-background: linear-gradient(to top, $upper-color, $upper-color) 100% 50% / 100% $track-height no-repeat transparent;
```
[the remark this answers is not in the document]
Comment (alexkhalevine) [reply src/components/RangeSlider/customTheme.scss]: As I showed you, it defines shape and color of slider bar

File: src/components/RangeSlider/customTheme.scss (hunk 1 above)
Comment (alexkhalevine) [reply src/components/RangeSlider/customTheme.scss]: nope, tested

File: src/components/RangeSlider/icons/icn-decrease-btn.jsx (hunk 2 above)
Comment (alexkhalevine) [reply src/components/RangeSlider/icons/icn-decrease-btn.jsx]: done

File: src/components/RangeSlider/icons/icn-increase-btn.jsx (hunk 3 above)
Comment (alexkhalevine) [reply src/components/RangeSlider/icons/icn-increase-btn.jsx]: done

File: src/components/RangeSlider/index.scss (hunk 4 above)
Comment (alexkhalevine) [reply src/components/RangeSlider/index.scss]: done

**Pull request merged**